### ISSUE TITLE
wont start up pls help

### ISSUE THREAD

**Issue by PAT899** [anegostudios/VintageStory-Issues#411]
```
Running on 64 bit Windows with 16 GB RAM 
Version: v1.12.5 (Stable)3/5/2020 2:41:15 AM: Critical error occurred
System.NullReferenceException: Object reference not set to an instance of an object.
   at Vintagestory.Client.NoObf.ClientPlatformWindows.LoadTexture(IBitmap bmp, Boolean linearMag, Int32 clampMode, Boolean generateMipmaps)
   at _mKeEWUbBZsjLS6VlGWgOoYQFk2K._KNs6g2GMtsTu5gnL4MXdRgZVlLh()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._HdDsdRJDH8nMF88r278C4eqdINGA()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._gjiBClIE7Yspuz65gCqPGSAnUYeb()
   at _lt8TgSVuBImgzFgAFlSeVzSFsSy._NbPATb9BXlxRc4ZhH2GgyFCUInp(Single )
   at Vintagestory.Client.NoObf.ClientPlatformWindows.window_RenderFrame(Object sender, FrameEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at OpenTK.GameWindow.RaiseRenderFrame(Double elapsed, Double& timestamp)
   at OpenTK.GameWindow.DispatchRenderFrame()
   at OpenTK.GameWindow.Run(Double updates_per_second, Double frames_per_second)
   at _hCuHXesCXlp0LGSdrsPFpw4up5n._mOm8EuKIvHpaJoztxNWRpAoOpbM(_55ISULxdcpPfF8D2JP831aWhOBe , String[] )
   at _iCfIbY24GyBXP5lAwqsSlD9COxC._mOm8EuKIvHpaJoztxNWRpAoOpbM(ThreadStart )
-------------------------------
```

**Comment by RudiTheStrange** [comment]
I see :D it's a great game even though it is still in early access

**Comment by PAT899** [comment]
love it was literally just playing like 3 hours ago

**Comment by Swarsulf** [comment]
i have the same error in my game, 
I hope that they will find the error quickly

**Comment by om3gaman92** [comment]
FIX ;)
https://www.vintagestory.at/blog.html/news/crash-on-startup-fix-v1126-r240/

**Comment by copygirl** [comment]
Duplicate of #407, fixed in `1.12.6`, so please update!
